.release/ci.hcl

```diff
@@ -13,6 +13,7 @@ project "boundary" {
 
     release_branches = [
       "main",
+      "release-mods"
```

Comment:
This will be removed before merging; it's just needed to test the full pipeline e2e